apps/sql-server/fdw/fdw/common.py

```diff
@@ -65,93 +87,121 @@ def get_schema() -> Dict:
 }
 
 
-def property_columns(data: dict) -> List[ColumnDefinition]:
-    """
-    Create a list of ColumnDefinitions for the given properties.
-    This is used to create the foreign table in PostgreSQL.
-    """
-    columns = []
-    if "properties" in data and data["properties"] is not None:
-        assert isinstance(data["properties"], dict), \
-            f"Expected properties to be a dict, got {type(data['properties'])}"
-        for prop, prop_data in data["properties"].items():
-            try:
-                count, indexed, type_ = prop_data
-                columns.append(ColumnDefinition(
-                    column_name=prop,
-                    type_name=TYPE_MAP[type_.lower()],
-                    options=ColumnOptions(count=count, indexed=indexed, type=type_.lower()).to_string()))
-            except Exception as e:
-                logger.exception(
-                    f"Error processing property '{prop}': {e}")
-                raise
-
-    # Add the _uniqueid column
-    columns.append(ColumnDefinition(
-        column_name="_uniqueid", type_name="text", options=ColumnOptions(count=data.get("matched", 0), indexed=True, unique=True, type="string").to_string()))
-
-    return columns
-
-
-class TableOptions(BaseModel):
-    """
-    Options passed to the foreign table from `import_schema`.
-    """
-    class_: Optional[str] = None  # class name of the entity or connection as reported by GetSchema
-    type: str = "entity"  # object type, e.g. "entity", "connection", "descriptor"
-    count: int = 0  # number of matched objects
-    # command to execute, e.g. "FindEntity", "FindConnection", etc.
-    command: str = "FindEntity"
-    # field to look for in the response, e.g. "entities", "connections"
-    result_field: str = "entities"
-    extra: dict = {}  # additional options for the command
-    blob_column: Optional[str] = None  # column containing blob data
-    # properties of the descriptor set, if applicable
-    descriptor_set_properties: Optional[dict] = None
-    # operation types for the descriptor set, if applicable
-    operation_types: Optional[List[str]] = None
+logger = logging.getLogger(__name__)
+
+
+class Curry:
```

Comment:
This seems interesting. This is to have operations in sequence, right?

Reply:
This is to permit serialization and deserialization of function pointers in text. I added a docstring on `Curry` that should make it a little clearer.